### ISSUE TITLE
Is there a way to make "Go To Definition" skip the .py and go straight to the ipynb?

### ISSUE THREAD

**Issue by Taytay** [fastai/nbdev-vscode#23]
Thanks again for this extension!

Probably the _biggest_ issue in my workflow with nbdev (not this extension, but nbdev) is that CMD+clicking on a variable (Go to Definition in vscode), takes me to the .py file instead of taking me to the .ipynb notebook. I will often go there, make edits, and only later realize I was editing the wrong thing. It bites me multiple times a day. 

Is it possible to make this extension create a new command that runs "go to definition" and then immediately runs "nbdev: navigate"? 

If so, it wouldn't be perfect, since it wouldn't take me to the perfect spot in the notebook cell, but it would be closer.

**Comment by mauroibzglobant** [comment]
I agree with this feedback. Now that our project has grown large, I find myself making small changes over the .py since it's faster, and then having to copy those changes back into the notebook, which became annoying and against the spirit of nbdev. Having easier access to the real source code would make this easier to avoid.
